### PR TITLE
Fix & Improve README

### PR DESCRIPTION
**Pull request by iagooar**
I found that your fork of https://github.com/xerions/phoenix_swagger is much more complete, thanks for it!

Unfortunately, it took me a lot more than would be needed to find out that the `README.md` file contains outdated information. I guess that it is a mix of the original `README` and your changes.

I have fixed the misleading information and improved a bit the grammar & style.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,76 +53,90 @@ def swagger_spec do
...
 
 The `swagger_path` macro takes two parameters:
 
 * The name of the controller action
-* `do` block containing calls into the PhoenixSwagger.Path module.
+* A `do` block containing calls into the `PhoenixSwagger.Path` module.
 
-The body of the DSL is only a thin layer of syntax sugar over a regular phoenix function pipeline.
-The example above can be re-written as:
+The body of the DSL is only a thin layer of syntax sugar over a regular `Phoenix` function pipeline.
```
Comment (mbuhot) [inline README.md:110]: @yagooar: Was this section describing the implementation of the DSL useful for you?

It was interesting to me as I was learning Elixir, but maybe it doesn't need to be in the README, just the API docs?

File: README.md [hunk 2]
```diff
@@ -59,76 +53,90 @@ def swagger_spec do
...
+The `parameter` option provides the description of a parameter for the given action and may take four positional arguments, and a keyword list of optional arguments:
 
-* The location of the parameter. Possible values are `query`, `header`, `path`, `formData` or `body`. [required];
 * The name of the parameter. [required];
-* The type of the parameter. Allowed only [swagger data types](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#data-types
+* The location of the parameter. Possible values are `query`, `header`, `path`, `formData` or `body`. [required];
+* The type of the parameter. Allowed [swagger data types](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#data-types
 ) [required];
-* Description of a parameter. Can be elixir's `String` or function/0 that returns elixir's string;
+* Description of a parameter. Can be an Elixir `String` or `function/0` that returns an Elixir string;
```
Comment (mbuhot) [inline README.md:135]: I'm not sure if the `function/0` option even works? 
We certainly don't use it that way internally - maybe just remove mention of it?

**Comment by mbuhot** [comment]
Thanks for the contribution @yagooar!

**Comment by iagooar** [comment]
Thanks for merging and keep up the good work! 👍

**Pull request merged**